sdk/android/Picovoice/picovoice/src/main/java/ai/picovoice/picovoice/Picovoice.java

```diff
@@ -340,7 +394,8 @@ public Picovoice build(Context appContext) throws PicovoiceException {
                         rhino,
                         inferenceCallback);
             } catch (PorcupineException | RhinoException e) {
-                throw new PicovoiceException(e);
+                mapToPicovoiceException(e);
```

Comment:
if you make this function return a PicovoiceException I think you can just do `throw mapToPicovoiceException(e);` and delete the return null